connectors/aws/aws-textract/pom.xml

```
@@ -0,0 +1,85 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.camunda.connector</groupId>
+    <artifactId>connector-aws-parent</artifactId>
+    <version>8.6.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <name>connector-aws-textract</name>
+  <description>Camunda AWS Textract Connector</description>
+  <artifactId>connector-aws-textract</artifactId>
+  <packaging>jar</packaging>
+
+  <licenses>
+    <license>
+      <name>Camunda Self-Managed Free Edition license</name>
+      <url>https://camunda.com/legal/terms/cloud-terms-and-conditions/camunda-cloud-self-managed-free-edition-terms/</url>
+    </license>
+    <license>
+      <name>Camunda Self-Managed Enterprise Edition license</name>
+    </license>
+  </licenses>
+
+  <properties>
+    <license.inlineheader>Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+      under one or more contributor license agreements. Licensed under a proprietary license.
+      See the License.txt file for more information. You may not use this file
+      except in compliance with the proprietary license.</license.inlineheader>
+  </properties>
```

I think, this is the indentation problem:
```suggestion
    <license.inlineheader>Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
under one or more contributor license agreements. Licensed under a proprietary license.
See the License.txt file for more information. You may not use this file
except in compliance with the proprietary license.</license.inlineheader>
  </properties>
```

---

I have checked 5 more License.txt files and everywhere same data

---

It should look like this: https://github.com/camunda/connectors/blob/4d95a3eaea838a93a7fbac14fbdab3ad2c6c7b50/connectors/kafka/pom.xml#L29 and https://github.com/camunda/connectors/blob/4d95a3eaea838a93a7fbac14fbdab3ad2c6c7b50/connectors/kafka/src/main/java/io/camunda/connector/kafka/outbound/KafkaConnectorFunction.java#L1. I think, the others are incorrect. We can fix those in separate PR.